services/distribution/src/test/java/app/coronawarn/server/services/distribution/statistics/KeyFigureCardFactoryTest.java

```diff
@@ -134,8 +135,9 @@ void shouldThrowAnExceptionIfAnyPropertyIsMissing() {
     @ValueSource(ints = {0, -1})
     void shouldFailIfInfectionCardIsNotValid(Integer value) {
       statisticsJsonStringObject.setInfectionsReportedCumulated(value);
+      final int cardOrdinal = INFECTIONS_CARD.ordinal();
       assertThatThrownBy(
-          () -> figureCardFactory.createKeyFigureCard(statisticsJsonStringObject, INFECTIONS_CARD.ordinal()))
+          () -> figureCardFactory.createKeyFigureCard(statisticsJsonStringObject, cardOrdinal))
```

Comment:
I don't get the point here... something about not being `final` within closure, I guess???

Reply:
Similar - There were more than one single call insise the Assertion: createKeyFigureCard(...) and ordinal()